<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GbsImageWidget</class>
 <widget class="QWidget" name="GbsImageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="9,1">
      <item>
       <widget class="QLabel" name="labImage">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="btnPrevious">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../main.qrc">
            <normaloff>:/image/左-三角.png</normaloff>:/image/左-三角.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labPage">
          <property name="text">
           <string>0/0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnNext">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../main.qrc">
            <normaloff>:/image/右-三角.png</normaloff>:/image/右-三角.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../main.qrc"/>
 </resources>
 <connections/>
</ui>
